src/runtime/server/services/local/getServerSession.ts

```diff
@@ -0,0 +1,51 @@
+import { type H3Event, createError } from 'h3'
+import getURL from 'requrl'
+import { joinURL } from 'ufo'
+import { jsonPointerGet, useTypedBackendConfig } from '../../../helpers'
+import { getToken } from './getToken'
+
+// @ts-expect-error - #auth not defined
+import type { SessionData } from '#auth'
+import { useRuntimeConfig } from '#imports'
+
+function joinPathToApiURL(event: H3Event, path: string) {
```

Comment:
nit: would prefer if this was down the file as it is a helper (consumers are more interested in reading `getServerSession`